PoiDetail

jcaliksygic opened the pull request:
AddressCell renamed to HeaderCell,
VC presentAsChildViewController for better understanding,
doc,
DataSources instead of ViewModels

jcaliksygic committed:
PoiDetail

AddressCell renamed to HeaderCell,
VC presentAsChildViewController for better understanding,
doc

jcaliksygic committed:
Merge branch 'feature/mvc_switch' of github.com:Sygic/sygic-ui-kit-ios into feature/poi-detail-mvc

jcaliksygic committed:
ActionButton mvc

deleted viewModel,
actionButton is configured by its properties

jcaliksygic committed:
CI test

marekdovjak commented on SygicUIKit/Classes/Views/BottomSheets/PoiDetail/PoiDetailHeaderCell.swift:
```diff
@@ -55,7 +55,7 @@ public class PoiDetailAddressCell: UITableViewCell {
 
  // MARK: - Public Methods
-    public func update(with viewModel: SYUIPoiDetailAddressDataSource) {
+    public func update(with viewModel: SYUIPoiDetailHeaderDataSource) {
```
viewModel premenovat

marekdovjak commented on SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift:
```diff
@@ -51,8 +50,8 @@ internal class SYUIPoiDetailView: UIView {
     public func reloadData() {
-        guard let viewModel = viewModel else { return }
-        actionButtonsView.update(with: viewModel.buttonsViewModel)
+        guard let viewModel = delegate else { return }
```
premenovat viewModel

jcaliksygic replied:
premenoval som tusim v brenchi k action buttonom

marekdovjak commented on SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift:
```diff
@@ -137,24 +136,24 @@ extension SYUIPoiDetailView: UITableViewDataSource {
     public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        guard let viewModel = viewModel, let section = SYUIPoiDetailSectionType(rawValue: section) else { return 0 }
-        if section == .address {
+        guard let viewModel = delegate, let section = SYUIPoiDetailSectionType(rawValue: section) else { return 0 }
```
viewmodel...

jcaliksygic replied:
premenovane v inej brenchi

Commit:
Try to fix travis ci build

Commit:
Specify another travis ci simulator

Commit:
Merge branch 'feature/mvc_switch' into feature/action-button-mvc

# Conflicts:
#	.travis.yml
#	SygicUIKit.podspec

jcaliksygic committed:
Merge branch 'feature/mvc_switch' of github.com:Sygic/sygic-ui-kit-ios into feature/poi-detail-mvc

jcaliksygic committed:
naming

jcaliksygic committed:
Merge branch 'feature/poi-detail-mvc' of github.com:Sygic/sygic-ui-kit-ios into feature/action-button-mvc

jcaliksygic committed:
ActionButton

public titleLabel, subtitleLabel that reference private customTitleLabel and customSubtitleLabel,
cleanup

jcaliksygic committed:
Merge pull request #3 from Sygic/feature/action-button-mvc

ActionButton mvc
